deltachat-ios/Chat/Views/Cells/VideoInviteCell.swift

```diff
@@ -0,0 +1,142 @@
+        label.paddingLeading = 4
+        label.paddingTrailing = 4
+        label.clipsToBounds = true
+        label.isAccessibilityElement = false
+        label.backgroundColor = UIColor(alpha: 200, red: 50, green: 50, blue: 50)
+        return label
+    }()
+
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: .subtitle, reuseIdentifier: reuseIdentifier)
+        clipsToBounds = false
+        backgroundColor = .none
+        setupSubviews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func setupSubviews() {
+        contentView.addSubview(videoIcon)
+        contentView.addSubview(avatarView)
+        contentView.addSubview(messageBackgroundContainer)
+        contentView.addSubview(messageLabel)
+        contentView.addSubview(openLabel)
+        contentView.addSubview(bottomLabel)
+        contentView.addConstraints([
+            videoIcon.constraintAlignTopTo(contentView, paddingTop: 12),
+            videoIcon.constraintCenterXTo(contentView, paddingX: -36),
+            avatarView.constraintAlignTopTo(contentView, paddingTop: 12),
+            avatarView.constraintCenterXTo(contentView, paddingX: 36),
+            messageLabel.constraintToBottomOf(videoIcon, paddingTop: 16),
+            messageLabel.constraintAlignLeadingMaxTo(contentView, paddingLeading: UIDevice.current.userInterfaceIdiom == .pad ? 150 : 50),
+            messageLabel.constraintAlignTrailingMaxTo(contentView, paddingTrailing: UIDevice.current.userInterfaceIdiom == .pad ? 150 : 50),
+            messageLabel.constraintCenterXTo(contentView),
+            openLabel.constraintToBottomOf(messageLabel),
+            openLabel.constraintCenterXTo(contentView),
+            openLabel.constraintAlignLeadingMaxTo(contentView, paddingLeading: UIDevice.current.userInterfaceIdiom == .pad ? 150 : 50),
+            openLabel.constraintAlignTrailingMaxTo(contentView, paddingTrailing: UIDevice.current.userInterfaceIdiom == .pad ? 150 : 50),
+            messageBackgroundContainer.constraintAlignLeadingTo(messageLabel, paddingLeading: -6),
+            messageBackgroundContainer.constraintAlignTopTo(messageLabel, paddingTop: -6),
+            messageBackgroundContainer.constraintAlignBottomTo(openLabel, paddingBottom: -6),
+            messageBackgroundContainer.constraintAlignTrailingTo(messageLabel, paddingTrailing: -6),
+            bottomLabel.constraintAlignTrailingTo(messageBackgroundContainer),
+            bottomLabel.constraintToBottomOf(messageBackgroundContainer, paddingTop: 8),
+            bottomLabel.constraintAlignLeadingMaxTo(contentView, paddingLeading: UIDevice.current.userInterfaceIdiom == .pad ? 150 : 50),
+            bottomLabel.constraintAlignBottomTo(contentView, paddingBottom: 12)
+        ])
+        selectionStyle = .none
+    }
+
+    func update(dcContext: DcContext, msg: DcMsg) {
+        messageLabel.text = msg.text
```

Comment:
the cell should not contain the text of the message but a dedicated string:

- for outgoing invites, use string `videochat_you_invited_hint`
- for incoming invited, use `videochat_contact_invited_hint` with the placeholder replaced by the contacts display name